VideoLocker/build.gradle

```diff
@@ -302,9 +302,10 @@ android {
     }
 
     lintOptions {
-        abortOnError false
-        lintConfig file("lint.xml")
-        disable 'MissingTranslation'
+        abortOnError true
+        showAll true
+        warning 'TypographyQuotes', 'InvalidPackage', 'ValidFragment', 'NewApi'
```

Comment:
Sort of?

It will set the severity of the listed issues to "warning" instead of whatever their default is.

So if the default was "ignore" (like it is for `TypographyQuotes`) it will now be a warning, so that it will show in the report. But if the default was `error` or `fatal` it will no longer be fatal, just a warning.